packages/form-js-editor/src/features/properties-panel/entries/NumberEntries.js

```diff
@@ -91,7 +91,12 @@ function NumberArrowStep(props) {
     return value;
   };
 
-  const setValue = (value) => editField(field, [ 'increment' ], value);
+  const clearLeadingZeroes = (value) => {
+    const trimmed = value.replace(/^0+/g, '');
+    return trimmed.startsWith('.') ? '0' + trimmed : trimmed;
```

Comment:
```suggestion
    return value.replace(/^0+/g, '0');
```

Reply:
I think this wouldn't work, because then 00005.25 => 05.25

Reply:
Yeah, you're right
We could update the regexp, but I think your solution is easier to read

Reply:
Actually there's a flaw in this one 😢, gonna push a new version just now. Basically value can be null I haven't accounted or tested for that. Let me just push the commit through.